.deepsource.toml

```diff
@@ -0,0 +1,8 @@
+version = 1
```

Comment:
Please add to the PR a description of how to use the `deepsource`.
How about adding it to the github actions?

Reply:
Hello sir i am a beginner in open source and i am  learning this first time so please sir guide me so i can do better

Reply:
It will make things easier if add to the github action

Reply:
Ok i will add How to use deepsource in PR Discription
 Thank you so much for guiding me

Reply:
Hello sir as per your guidence i have updated the changes on how to use DeepSource
Please tell me know if i can do anything else 
Thank you